SkylineToolsStore/src/org/labkey/skylinetoolsstore/SkylineToolsStoreModule.java

```diff
@@ -91,6 +92,7 @@ public void doStartup(ModuleContext moduleContext)
     {
         // add a container listener so we'll know when our container is deleted:
         ContainerManager.addContainerListener(new SkylineToolsStoreContainerListener());
+        SecurityManager.registerAllowedConnectionSource("script-src", "https://code.jquery.com/ui/1.13.2/jquery-ui.min.js");
```

Comment:
The key is primarily for use cases where the value may change due to config from users. I'll add some JavaDoc separately to help clarify.

```suggestion
        SecurityManager.registerAllowedConnectionSource("jquery-ui", "https://code.jquery.com/ui/1.13.2/jquery-ui.min.js");
```